lib/auth/firebaseAdmin.ts

```diff
@@ -32,9 +32,13 @@ export const getUserFromToken = async (
     });
 
   if (uid) {
-    const user = await getTalentById(uid);
-    console.log('found user: ', user);
-    return user;
+    try {
+      const user = await getTalentById(uid);
+      console.log('found user: ', user);
+      return user;
+    } catch (e) {
+      console.error(e);
```

Comment:
could you add eslint-ignore-line no-console (I think that's the right command) to this and all other console logs that will stay in the code for the time being, and remove all logs that only are there for checking purposes?

Reply:
done